scripts/check-third-party-license-file-paths

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for i in `grep -oP "(?<=Files: |\s{7})([^\*].*/.*)" ../doc/THIRD-PARTY-LICENSES`
+do
+	echo -n "../$i "
+	eval ls -d "../$i" 2> /dev/null | wc -l
```

Comment:
@markus2330 you mentioned that this script produces false positives?
How exactly? I was not able to reproduce any false positives.

Reply:
I think it was related to wildcards.

Reply:
Alright!

The supported formatting is still quite limited, but I guess we can keep the non-path results to a minimum.